screens/AbooutUs.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { View, Text, Image, FlatList, StyleSheet } from 'react-native';
+
+const TeamMembers = [ 
+  { 
+    id: "1", 
+    title: "Dr. Heng Sovannarith",
+    image: require('../assets/team/gallery.png'),
+  }, 
+  { 
+    id: "2", 
+    title: "Yem Hea",
+    image: require('../assets/team/yem_hea.jpg'),
+  }, 
+  { 
+    id: "3", 
+    title: "Sok Leaphea",
+    image: require('../assets/team/gallery.png'),
+  }, 
+  { 
+    id: "4", 
+    title: "Phon Sobon",
+    image: require('../assets/team/phon_sobon.jpg'),
+  }, 
+  { 
+    id: "5", 
+    title: "Tes Sophanna",
+    image: require('../assets/team/tes_sophanna.jpg'),
+  }, 
+  { 
+    id: "6", 
+    title: "Teok Yoteav",
+    image: require('../assets/team/gallery.png'),
+  }, 
+  { 
+    id: "7", 
+    title: "Yoem Sreypov",
+    image: require('../assets/team/gallery.png'),
+  }, 
+  { 
+    id: "8", 
+    title: "Uth Narin",
+    image: require('../assets/team/uth_narin.jpg'),
+  }, 
+  { 
+    id: "9", 
+    title: "Chet Chakriya",
+    image: require('../assets/team/chet_chakriya.jpg'),
+  }, 
+  { 
+    id: "10", 
+    title: "Pho Kakada",
+    image: require('../assets/team/pho_kakada.jpg'),
+  }, 
+  { 
+    id: "11", 
+    title: "Chea Menglim",
+    image: require('../assets/team/chea_menglim.jpg'),
+  }, 
+]; 
+
+const Item = ({ title, image }) => { 
+  return ( 
+    <View style={styles.item}> 
+      {image && <Image source={image} style={styles.image} />} 
+      <Text style={styles.title}>{title}</Text> 
+    </View> 
+  ); 
+};
```

Comment:
Should be a profile component

Reply:
I will make it to components later because I try to understand the structure first